src/components/button.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
```suggestion
import React, { FC } from 'react';
```
Not super important, but we always destructed like this in the last project